src/content/en/2024/sustainability.md

```diff
@@ -2,17 +2,1265 @@
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRQLDsXdZj62xe68w716gen0rQvuuGhXPAOSwdWwYjSBZf9BgJgEb-dp1Z_jB_Lp5YMsfH0FiNKwzDb/pubchart?oid=1021559874&format=interactive",
+  sheets_gid="1333464158",
+  sql_file="ssg_bytes_per_type_2022.sql"
+) }}
+
+It appears evident that site generators have prioritized optimization as part of their builds. Astro, Hugo, and Jekyll all have emissions substantially lower than all the CMS that have been categorized.
+
+These insights underscore the significant role that platforms and frameworks play in promoting sustainable web development. By setting effective defaults, platform creators and framework developers enable the construction of environmentally friendly websites right from the start.​
+
+These exceptions belong to the static site generator category, particularly Hugo and Jekyll, which typically support sites focused on blogs and text-heavy content with minimal JavaScript usage. Static site generators are also often chosen for their performance benefits, suggesting a greater likelihood of optimization beyond the norms of standard CMS-driven sites.
+
+For further information, refer to:
+- The WSGs 1.0 - SC 3.7, 3.21, and 5.28.
+
+## Conclusion
+
+Sustainability is a rapidly evolving and firmly established field that will increasingly have a seat at the table for those creating products and services for the Web. As with accessibility, legislation is helping to drive the need for conformance and the benefits it can bring to both people and the planet. By considering how you create websites and applications, the ethical decisions behind design and development, taking into account industry best practices, and tackling the digital variables that can have real-world impacts, each one of us can become sustainability advocates.
+
+Starting from the 2022 chapter, we realized that things are moving really fast for web sustainability and we are faced with many challenges:
+- There is more and more information out there and it might get difficult to follow: new resources and tools, existing stuff being updated, etc.
+- It appears essential to keep an eye on emerging technologies (metaverse faded away as fast as it appeared, AI is everywhere), since all of them have environmental impacts (and possibly benefits too). Solutionism is a growing risk:
+  - Offering digital solutions to fight climate change without considering their own impacts.
+  - Silver bullet offerings to reduce the environmental impacts of digital.
+- Greenwashing is everywhere, even with the best intention in mind.
+
+In the end, we see a generally wider adoption of technical best practices but still an increase of emissions. Efficiency is mandatory but sobriety and frugality are the way to go.
+
+### Actions you can take
+
+{# TODO - add intro paragraph #}
```

Comment:
@burakguneli I think this is what @lebreRafael wanted you to help with?

Reply:
@lebreRafael I added the intro paragraph to the google docs: https://docs.google.com/document/d/1AB_e7VaHPC307NycTXxwndOTgbFJld2_RqWm8abps1w/edit?disco=AAABXuHfL9Q

I don't know much about markdown so I didn't want to touch it. Would it be okay for you to review it and add it to the PR if everything looks good?